package.json

```diff
@@ -48,7 +48,7 @@
     "test": "npm run test:tap && npm run test:node-fetch && npm run test:fetch && npm run test:jest && tsd",
     "test:node-fetch": "node scripts/verifyVersion.js 16 || mocha test/node-fetch",
     "test:fetch": "node scripts/verifyVersion.js 16 || tap test/fetch/*.js",
-    "test:jest": "jest test/jest/test",
+    "test:jest": "jest test/jest/*.js",
```

Comment:
@mcollina found there was a test file for jest, that had been disabled, and by changing this and it running, we actually see this error, and with this fix its resolved